mt-kafka-mdm-sniff panics with sigsegv

how to reproduce:
* spin up a new mt cluster (`docker-cluster-query` in my tests)
* feed some fakemetrics into its kafka
* exec into one of the metrictank containers
* sniff the kafka data
* within seconds it exits

```
replay@mst-nb:~/go/src/github.com/grafana/metrictank/build$ docker exec -it bd6921ee0da9 sh
/ # mt-kafka-mdm-sniff
2019-12-05 16:28:24.971 [INFO] kafkamdm: available partitions [0 1 2 3 4 5 6 7]
2019-12-05 16:28:24.974 [INFO] kafkamdm: consumer created without error
2019-12-05 16:28:24.977 [INFO] kafkamdm: consuming from mdm:1 from offset 0
2019-12-05 16:28:24.977 [INFO] kafkamdm: consuming from mdm:2 from offset 0
2019-12-05 16:28:24.978 [INFO] kafkamdm: consuming from mdm:3 from offset 0
2019-12-05 16:28:24.978 [INFO] kafkamdm: consuming from mdm:0 from offset 0
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x942e57]

goroutine 52 [running]:
github.com/grafana/metrictank/input/kafkamdm.(*KafkaMdm).trackStats(0xc000176cc0, 0xc0000a1099, 0x3, 0x1)
        /go/src/github.com/grafana/metrictank/input/kafkamdm/kafkamdm.go:363 +0x327
created by github.com/grafana/metrictank/input/kafkamdm.(*KafkaMdm).consumePartition
        /go/src/github.com/grafana/metrictank/input/kafkamdm/kafkamdm.go:278 +0x1c0
```

--- a/input/kafkamdm/kafkamdm.go
+++ b/input/kafkamdm/kafkamdm.go
@@ -360,7 +360,9 @@ func (k *KafkaMdm) trackStats(topic string, partition int32) {
 			kafkaStats.Lag.Set(lag)
 			k.lagMonitor.StoreOffsets(partition, currentOffset, newest, ts)
 			kafkaStats.Priority.Set(k.lagMonitor.GetPartitionPriority(partition))
-			kafkaStats.Ready.Set(cluster.Manager.IsReady())
+			if cluster.Manager != nil {
+				kafkaStats.Ready.Set(cluster.Manager.IsReady())
+			}
 		}
 	}
 }

